bin/update_reference_document.sh

```diff
@@ -9,6 +9,7 @@ JSON_FILE="spec/fixtures/reference_document_parsed.json"
 HTML_EXPORT_FILE="spec/fixtures/reference_document_exported.html"
 AMP_EXPORT_FILE="spec/fixtures/reference_document_exported.amp.html"
 FACEBOOK_EXPORT_FILE="spec/fixtures/reference_document_exported.facebook.html"
+APPLE_NEWS_EXPORT_FILE="spec/fixtures/heading_and_basic_text_exported.apple_news.html"
```

Comment:
```suggestion
APPLE_NEWS_EXPORT_FILE="spec/fixtures/reference_document_exported.apple_news.json"
```

Reply:
Amended to:
`...reference_document_exported.apple_news.json`